packages/channels/package.json

```diff
@@ -25,13 +25,15 @@
     "@types/uuid": "^8.3.4"
   },
   "dependencies": {
+    "@slack/bolt": "^3.9.0",
     "axios": "^0.24.0",
     "cli-color": "^2.0.1",
```

Comment:
That's the new packages that is recommended to use. It's an official slack package